src/content/guides/code-splitting.md

```diff
@@ -265,7 +265,18 @@ Entrypoint index = index.bundle.js
     + 1 hidden module
 ```
 
-As `import()` returns a promise, it can be used with [`async` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). However, this requires using a pre-processor like Babel and the [Syntax Dynamic Import Babel Plugin](https://babeljs.io/docs/plugins/syntax-dynamic-import/#installation). Here's how it would simplify the code:
+As `import()` returns a promise, it can be used with [`async` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). However, this requires using a pre-processor like Babel and the [Syntax Dynamic Import Babel Plugin](https://babeljs.io/docs/plugins/syntax-dynamic-import/#installation). For `async await` `babel-polyfill` is required. Common usage can be found [here] https://babeljs.io/docs/en/babel-polyfill/. Or in case you are using mutiple entries. 
+
+Your webpack.config.js entry can look like: 
+
+```
```

Comment:
this isnt a valid config example, please be consistent with the rest of examples